bazel/abseil.patch

```diff
@@ -40,3 +40,16 @@ index 230bf1e..6e1b9e5 100644
 
 
  // ABSL_OPTION_USE_INLINE_NAMESPACE
+diff --git a/absl/flags/commandlineflag.h b/absl/flags/commandlineflag.h
```

Comment:
Is this being upstreamed?

Reply:
I'm negotiating with the absl team about that or pragma changes yep